### PR TITLE
allow empty fake arm cfg to populate with default

### PR DESCRIPTION
**Pull request by nfranczak**

### CODE REVIEW

File: etc/configs/fake.json [hunk 1]
```diff
@@ -7,10 +7,7 @@
         {
             "name": "arm1",
             "type": "arm",
-            "model": "fake",
```
Comment (biotinker) [inline etc/configs/fake.json:10]: Will this pass tests? The tests that use `fake` rely on it having the actual fake mockup arm, right?

File: components/arm/fake/fake.go [hunk 2]
```diff
@@ -105,7 +100,7 @@ func NewArm(cfg config.Component, logger golog.Logger) (arm.LocalArm, error) {
 	case modelPath != "":
 		model, err = referenceframe.ModelFromPath(modelPath, cfg.Name)
 	default:
-		err = errAttrCfgMissing
+		model = referenceframe.NewSimpleModel(cfg.Name)
```
Comment (biotinker) [inline components/arm/fake/fake.go:103]: Add a comment saying "if no arm model is specified, an empty one with 0 dof and 0 spatial transformation is created" or something along those lines.

File: etc/configs/fake.json (hunk 1 above)
Comment (nfranczak) [reply etc/configs/fake.json:10]: to confirm I did cmf+shift+f on VScode. This allows me to search the entire RDK.
no tests use `etc/configs/fake.json`

Comment (biotinker) [reply etc/configs/fake.json:10]: If the tests are happy with this change then I am too :)

**Review by biotinker — APPROVED** [review]
LGTM if tests pass

**Pull request merged**